### PR TITLE
Mobile navigation bar with sharing buttons

### PR DESCRIPTION
**Pull request by garfieldduck**
- add facebook, twitter sharing buttons
- add Line sharing button (mobile only)
- add article title for Twitter sharing
- use webpack to import svg images
- remove unnecessary sass prefix

### CODE REVIEW

File: src/components/navigation/DesktopNavBar.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+import fbIcon from '../../../static/asset/fb.svg'
+import logoIcon from '../../../static/asset/logo-desk.svg'
+import logoIconDark from '../../../static/asset/logo-desk-dark.svg'
+import React, { Component } from 'react'
+import smallLogo from '../../../static/asset/navbar-fixed-top-logo.svg'
 import styles from './DesktopNavBar.scss'
-import { FacebookButton, TwitterButton } from 'react-social'
-import { getAbsPath } from '../../lib/url-transformer'
-import { appId, donatePath } from '../../lib/constants'
+import twitterIcon from '../../../static/asset/twitter.svg'
```
Comment (nickhsine) [inline src/components/navigation/DesktopNavBar.js]: sorting?

File: src/components/navigation/MobileNavBar.js [hunk 2]
```diff
@@ -1,30 +1,70 @@
...
-  require('../../containers/NavBar.css')
-}
+import styles from './MobileNavBar.scss'
+import SubNavBar from './SubNavBar'
+import twitterIcon from '../../../static/asset/twitter.svg'
+import lineIcon from '../../../static/asset/line.svg'
 
-export default class MobileNaviBar extends Component {
+export default class MobileNavBar extends Component {
   constructor(props) {
```
Comment (nickhsine) [inline src/components/navigation/MobileNavBar.js]: ```constructor(props, context)``` ?

File: src/components/navigation/DesktopNavBar.js [hunk 3]
```diff
@@ -73,24 +70,32 @@ export default class DesktopNavBar extends Component {
...
     let burgerIconClass = styles.navIcon
+    if (this.state.open) {
+      burgerIconClass = classNames(styles.navIcon, styles.iconOpen)
+    }
+
+    let burgerMenu = <div className={burgerIconClass} onClick={()=> { this.setState( { open: !this.state.open } )}}>
+                        <span></span>
+                        <span></span>
+                        <span></span>
+                        <span></span>
```
Comment (nickhsine) [inline src/components/navigation/DesktopNavBar.js]: why empty ```span``` ?

File: src/components/navigation/MobileNavBar.js [hunk 4]
```diff
@@ -1,30 +1,70 @@
...
+        </div>
+        {burgerMenu}
+      </div>
+    )
+  }
+
+  _renderAritcleSecond(burgerMenu, searchBox, cUrl) {
+    const navItemClass = styles.navButton
+    const { pageTitle } = this.props
+    const lineUrl = `http://line.naver.jp/R/msg/text/?${encodeURI(pageTitle)}%0D%0A${encodeURI(cUrl)}`
```
Comment (nickhsine) [inline src/components/navigation/MobileNavBar.js]: ```encodeURI``` all the string?

**Comment by nickhsine** [comment]
👍 💯 🚢 :shipit:

File: src/components/navigation/DesktopNavBar.js (hunk 3 above)
Comment (garfieldduck) [inline src/components/navigation/DesktopNavBar.js]: for the burger menu icon

**Pull request merged**